start_integrate_test.sh

```diff
@@ -78,10 +78,9 @@ array+=("compatibility/rpc/triple/msgpack")
 array+=("compatibility/rpc/triple/pb/dubbogo-grpc")
 #array+=("compatibility/rpc/grpc")
 array+=("compatibility/rpc/jsonrpc")
-array+=("compatibility/rpc/triple/pb2")
```

Comment:
"compatibility/rpc/triple/pb2" testing part is for compatibility use(i.e. Old API). Please do not remove it.